chat-core/src/EmbeddedContent.ts

```diff
@@ -0,0 +1,90 @@
+import { PersistedPage } from "./Page";
+
+export interface EmbeddedContent {
```

Comment:
i think we're also adding `score` per https://github.com/mongodb/docs-chatbot/pull/19/files#diff-e4eb4230c061137b41887df64b518d22d061c49648bc27b90621b206dc08faadR115

Reply:
Done (but not in the actual EmbeddedContent type) --> https://github.com/mongodb/docs-chatbot/pull/19/commits/7ebd62ffd9683b8538e4f32595538a9b7560d817